modules/tanka/Makefile

```diff
@@ -12,5 +12,5 @@ tanka/fmt-test: satoshi/check-deps
 	make k8s/tanka/fmt-test
 
 ## Generate manifests using tanka
-tanka/generate: satoshi/check-deps
+tanka/generate: satoshi/check-deps jsonnet/install
```

Comment:
I believe this file is deprecated and since moved into https://github.com/mintel/build-harness-extensions/blob/main/modules/k8s/Makefile.tanka

@barrymars I can't remember if these old targets are still used (if they are, we can probably just ditch them and update any calling code now?)

Reply:
yeah these targets were just left in for backwards compatibility and proxy to `k8s/tanka/*`

thought i'd added a comment to that effect, but guess not